website/client/src/components/groups/inviteModal.vue

```diff
@@ -29,7 +29,7 @@
             :class="{
               'input-valid': invite.valid, 'is-invalid input-invalid': invite.valid === false}"
             @keyup="expandInviteList"
-            @change="checkInviteList"
+            @input="checkInviteList(); clearErrors(invite);"
```

Comment:
👍

Reply:
Yeah, would have done it that way myself, but the problem is that we only want to clear the error for this particular invite and not all of them. Inside `checkInviteList` method there is no handle to a particular invitation, it operates on the whole list. Of course could change the method signature so that we pass in the invitation, but I think that defeats the purpose since it's not actually required for the main functionality of that method.

Reply:
Ok, I made a new method `inviteUpdated(invite)` that acts as the event handler for `@input` and it will then call `clearErrors` and `checkInviteList`. That way the html reads nicer and also SRP :+1: